packages/language/src/language-server/code-actions/apply-quick-fixes.ts

```diff
@@ -82,25 +82,58 @@ export async function quickFixResolveInclude(
     diagnostics: [diagnostic],
     command: {
       title: "Apply INCLUDE fix",
-      command: "pli.applyIncludeFix",
+      command: "pli.applyQuickFixResolveInclude",
       arguments: [procGrpsFileUri.toString(), newContent],
     },
   };
 
   return action;
 }
 
+export async function quickFixCreateConfig(
+  diagnostic: Diagnostic,
+): Promise<CodeAction | undefined> {
+  const entryUri = UriUtils.basename(URI.parse(diagnostic.data.entryUri));
+  if (!entryUri) {
+    return;
+  }
+  const action: CodeAction = {
+    title: `Create a '.pliplugin' folder in the project root using the current file as the entry point in 'pgm_conf.json'`,
+    kind: CodeActionKind.QuickFix,
+    diagnostics: [diagnostic],
+    command: {
+      title: "Create configuration folder",
+      command: "pli.applyQuickFixCreateConfig",
+      arguments: [entryUri],
+    },
+  };
+
+  return action;
+}
+
 export async function applyQuickFixes(
   diagnostics: Diagnostic[],
 ): Promise<CodeAction[] | undefined> {
   const actions: CodeAction[] = [];
   // PLI CODES LIST
   const CODE_UNRESOLVED_INCLUDE = "IBM3841IS"; // The INCLUDE file could not be found, or if found, it could not be opened.
+  const CODE_MISSING_CONFIG = "_TB002E";
```

Comment:
Suggestion: Please don't refer to the same constants multiple times as strings.
```suggestion
  const CODE_MISSING_CONFIG = Internal.DiagnosticMissingConfiguration.code;
```

Reply:
Good catch, thanks